turingschool-examples/hang-in-there-boilerplate
# Iteration 2 input forms

### What’s this PR do?

* Added the Iteration 2 which includes 
 
1.  submit data into images titles and quotes arrays 

2. uses the values from submission to make a new object instance of the poster class 

3.  redirects back to main poster view and hiding the form 

4. displays the newly created poster in the main view. 

#### Where should the reviewer start?

* main. js 

* Then by click in the HTML browser 

#### How should this be manually tested?

* clicking on the make your own poster, then entering in the 3 values, then clicking show my poster button and seeing verifying if that poster shows up 

* After doing that without refreshing, clicking the show random poster until the values show up in the display. 

* In the HTML browser page makes a new poster doing the step in the 1st bullet up until clicking show my poster, open the console, and entering the console.log(madePoster) verifying that the console.log values are the same as displayed after clicking the show my poster button. 

#### Any background context you want to provide?

* during this pull request we refactored our solution to iteration 0 in order to create a random poster object instance of the poster class. 

#### What are the relevant tickets?

* update README file will be updated in the next branch.
